src/pages/settings/project/ProjectSettings.jsx

```diff
@@ -109,6 +109,8 @@ const ProjectSettings = () => {
     deleteProject(projectID)
       .then(({ queued, newProjects }) => {
         if (!queued) {
+          if (localStorage.getItem("lastOpenedProject") === projectID) localStorage.removeItem("lastOpenedProject")
```

Comment:
No need to check this actually, since the project going to be deleted would be the last opened project as well